### PR TITLE
Swf delay import

### PR DESCRIPTION
**Pull request by blink1073**
Prevent `_swf.py` from loading unless needed.

### CODE REVIEW

File: imageio/plugins/swf.py [hunk 1]
```diff
@@ -214,6 +220,8 @@ def _get_meta_data(self, index):
     class Writer(Format.Writer):
         
         def _open(self, fps=12, loop=True, html=False, compress=False): 
+            if not swf:
```
Comment (almarklein) [inline imageio/plugins/swf.py]: `_swf`

**Comment by almarklein** [comment]
Can we use either do:
* use `lib()` everywhere; or
* do a check in `open` whether `_swf` exists and call `_load_lib()` if necessary. Use `_swf` everywhere else.

I'm leaning towards the latter. We should make the behavior consisten for dicom, swf and tiffile.

**Comment by blink1073** [comment]
How does that look?

**Comment by almarklein** [comment]
Great!  Do you want to do the same for dicom? If it suits you can can make the change in this PR...

**Comment by blink1073** [comment]
Let's keep them separate.

**Comment by almarklein** [comment]
Ok, thanks!

**Pull request merged**